src/transform.py

```diff
@@ -209,33 +333,45 @@ def publishable_speakers(accepted_proposals: set[str]) -> dict[str, PretalxSpeak
     return output
 
 
-def save_publishable_sessions():
+def save_publishable_sessions(publishable: dict[str, PretalxSubmission]):
     path = Config.public_path / "sessions.json"
 
-    publishable = publishable_submissions()
+    for sub in publishable.values():
+        if sub.start is not None:
+            PretalxSubmission.set_talks_in_parallel(sub, publishable)
+            PretalxSubmission.set_talks_after(sub, publishable)
+            PretalxSubmission.set_talks_before(sub, publishable)
 
     data = {k: v.model_dump() for k, v in publishable.items()}
     with open(path, "w") as fd:
         json.dump(data, fd, indent=2)
 
 
-def save_publishable_speakers():
+def save_publishable_speakers(publishable: dict[str, PretalxSubmission]):
     path = Config.public_path / "speakers.json"
 
-    publishable = publishable_submissions()
     speakers = publishable_speakers(publishable.keys())
 
     data = {k: v.model_dump() for k, v in speakers.items()}
     with open(path, "w") as fd:
         json.dump(data, fd, indent=2)
 
 
+def save_all():
+    if not Config.public_path.exists():
+        Config.public_path.mkdir(parents=True)
+
+    publishable = publishable_submissions()
+    save_publishable_sessions(publishable)
+    save_publishable_speakers(publishable)
+
+
 if __name__ == "__main__":
+    print(f"Transforming {Config.event} data...")
     print("Checking for duplicate slugs...")
     assert len(set(s.slug for s in publishable_submissions().values())) == len(
         publishable_submissions()
     )
```

Comment:
We should probably handle the submissions with the same names, instead of complaining about it.